Modules/ZDC/src/ZDCRawDataCheck.cxx

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   ZDCRawDataCheck.cxx
+/// \author My Name
+///
+
+#include "ZDC/ZDCRawDataCheck.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/Quality.h"
+#include "QualityControl/QcInfoLogger.h"
+// ROOT
+#include <TH1.h>
+
+#include <DataFormatsQualityControl/FlagReasons.h>
+
+using namespace std;
+using namespace o2::quality_control;
+
+namespace o2::quality_control_modules::zdc
+{
+
+void ZDCRawDataCheck::configure() {}
+
+Quality ZDCRawDataCheck::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
```

Comment:
If this check is not actually doing anything meaningful, I would recommend to remove it.

Reply:
I did not yet implemented the check of histograms. I created the checker for raw histograms because we had some histograms that we want use for the shifter. I wanted try if the json file configuration was correct and if when run the task runned also the checker.  If you are agree i prefer don't remove it,because in the next commites i'll implement it.

Reply:
Sure, no problem. Just FYI, if you removed the check from the json, it should still run the task and store the objects.